pkg/network/sdn-cni-plugin/openshift-sdn.go

```diff
@@ -213,11 +213,23 @@ func (p *cniPlugin) CmdAdd(args *skel.CmdArgs) error {
 					},
 					Gw: defaultGW,
 				}
-				err = netlink.RouteAdd(route)
-				if err != nil {
+				if err := netlink.RouteAdd(route); err != nil {
 					return fmt.Errorf("failed to configure macvlan device: %v", err)
 				}
 			}
+
+			// Add a route to service network via SDN
+			_, serviceIPNet, err := net.ParseCIDR(config.ServiceNetworkCIDR)
+			if err != nil {
+				return fmt.Errorf("failed to configure macvlan device: %v", err)
```

Comment:
This doesn't really matter much since it should be "can't happen", but, `"failed to parse ServiceNetworkCIDR: %v"`